dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicET.java

```diff
@@ -1308,4 +1308,22 @@ public void testManualPublishCustomDockerPathRegistry() {
                         "master", "--toolname", "alternate", "--private", "true", "--tool-maintainer-email", "duncan.andrew.g@gmail.com", "--script" });
 
         }
+
+        /**
+        * This tests that you can refresh user data by refreshing a tool
+         * ONLY WORKS if the current user in the database dump has no metadata, and on Github there is metadata (bio, location)
```

Comment:
Looks like the test will pass as long as the current user's metadata was not "location='Toronto' and bio='I am a test user"